test/Push.hs

```diff
@@ -1,14 +1,19 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This shouldn't be required. It's the one extension I've put in the cabal file (although I'm considering removing it for consistency and better tooling support).